examples/notebooks/persistent_dataset_speed.ipynb

```diff
@@ -464,4 +463,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
+}
```

Comment:
Please open the file in jupyter notebook and save it, otherwise, it will leave this "no new line" error.
Thanks.

Reply:
Sorry, I did not know about this problem. I hope 2a5cddd1b33b033974f6d6953487db1a9993780f solves it. Cheers